docs/reference/indices/index-templates.asciidoc

```diff
@@ -53,16 +66,31 @@ template for the `logs-*` index pattern, assign your template a priority of
 for `logs-*-*`.
 ====
 // end::built-in-index-templates[]
+****
 
-When a composable template matches a given index
-it always takes precedence over a legacy template. If no composable template matches, a legacy
-template may still match and be applied.
+[discrete]
+[[create-index-templates]]
+== Create composable index templates
 
```

Comment:
I think the focus needs to be on creating an index template. Since you want to highlight the Runtime field mapping, it might be easier to frame it as defining it directly in the index template, rather than in a component template for the UI piece.